Automattic/wp-calypso
# Checkout: update checkout styles to match Calypso

#### Changes proposed in this Pull Request

Since launching checkout, there have been some subtle changes to our Calypso styles. In this PR, I'm making some updates to sync up the colours and button copy text weight. 

**Before**
![image](https://user-images.githubusercontent.com/6981253/100623593-5edcca80-32f0-11eb-8612-960b010e424f.png)


**After**
![image](https://user-images.githubusercontent.com/6981253/100623511-453b8300-32f0-11eb-824a-084ce4d63796.png)


#### Testing instructions



* Add a product to your cart and go to checkout.
* Confirm:
 * That nothing looks broken or out of place.
 * Blue looks brighter
 * Button text weight is lighter (not bold anymore).

Note: The apply coupon button position will be fixed in this PR: https://github.com/Automattic/wp-calypso/pull/47809
